control-statistics/src/main/java/fi/nls/oskari/control/statistics/GetIndicatorListHandler.java

```diff
@@ -78,7 +78,7 @@ JSONObject getIndicatorsListJSON(long datasourceId, User user, String language)
     }
 
     private Optional<JSONObject> toJSON(StatisticalIndicator indicator, String language) {
-        if(indicator == null || indicator.getLayers() == null || indicator.getLayers().isEmpty()) {
+        if(indicator == null) {
```

Comment:
if `indicator.getLayers()` returns `null` then the whole `toJSON(StatisticalIndicator indicator, String language)` throws and catches the NullPointerException and still returns `Optional.empty()`. It might be that that `StatisticalIndicator#getLayers` never returns null (maybe an empty collection is stored in the database always) but that's not something I'd trust.